storage/database.go

```diff
@@ -198,9 +208,14 @@ func (d *db) Open() error {
 	}
 	d.state = databaseOpen
 
-	// All goroutines must be accounted for with dbOngoingTasks to receive done signal
+	// Goroutines must be accounted for with dbOngoingTasks in order to receive done signal
 	go d.reportLoop()
 	go d.ongoingTick()
+
+	// Explicitly start the repairer in Open and stop it in Close so there is no need to
+	// register this goroutine with dbOngoingTasks
+	go d.repairer.Start()
```

Comment:
I wanted it to be explicit w.r.t. whether a goroutine is spawned. I tried to follow the same pattern as other components but couldn't find any with Start/Stop. That said, I don't feel too strongly either way.